htmd/parameterization/cli.py

```diff
@@ -51,20 +51,28 @@ def cli_parser():
                         default=None, dest="freezeq")
     parser.add_argument("--no-geomopt", help="Do not perform QM geometry optimisation when fitting torsions (default: "
                                              "%(default)s)", action="store_false", dest="geomopt", default=True)
+    parser.add_argument("--seed", help="Random number generator seed (default: %(default)s)", type=int,
```

Comment:
Is this useful for the user? I would add `help=argparse.SUPPRESS`

Reply:
Yes, it is!

Reply:
As in, if a parameterization fails, they could try a different seed?

Reply:
Exactly! :)